### PR TITLE
Issues/251

### LINKED ISSUE #251: Document datetime settings for PHP apps

### PR DESCRIPTION
**Pull request by divarvel**
Closes #251

### CODE REVIEW

File: runtimes/php/php-apps.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Please note the absolute path style: `/public`.
 ### Change PHP settings
 
 Most of PHP settings can be changed using a `.user.ini` file. If you want it to be used for any file and you set a different `webroot` than `/`, you need to put this file in that directory instead of the root of your repository. Please refer to the
-[official documentation](http://www.php.net/manual/en/configuration.file.per-user.php) for more information.
+[official documentation](http://www.php.net/manual/en/configuration.file.per-user.php) for more information. You can review the [available directives](http://www.php.net/manual/en/ini.list.php), all the `PHP_INI_USER`, `PHP_INI_PERDIR`, and `PHP_INI_ALL` directives can be set from within `user.ini`.
```
Comment (urcadox) [inline runtimes/php/php-apps.md]: s/, all/; all/ :smile:

File: runtimes/php/php-apps.md [hunk 2]
```diff
@@ -447,6 +447,15 @@ Laravel doesn't need Monolog to retrieve logs via Clever console or Clever CLI.
 
 Then, set `APP_LOG=syslog` as Clever application environment variable.
 
+## Timezone configuration
+
+All instances on Clever Cloud run on the UTC timezone. We recommend to handle all your dates in UTC internally, and only handle timezones when reading or displaying dates. You can set PHP's time zone setting with `.user.ini`.
```
Comment (urcadox) [inline runtimes/php/php-apps.md]: I would put "You can set...." in another paragraph and add something like 'However,".

File: runtimes/php/php-apps.md (hunk 1 above)
Comment (divarvel) [reply runtimes/php/php-apps.md]: done

File: runtimes/php/php-apps.md (hunk 2 above)
Comment (divarvel) [reply runtimes/php/php-apps.md]: done, not sure about the "however", they are not opposed imo

File: runtimes/php/php-apps.md [hunk 3]
```diff
@@ -447,6 +447,16 @@ Laravel doesn't need Monolog to retrieve logs via Clever console or Clever CLI.
 
 Then, set `APP_LOG=syslog` as Clever application environment variable.
 
+## Timezone configuration
+
+All instances on Clever Cloud run on the UTC timezone. We recommend to handle all your dates in UTC internally, and only handle timezones when reading or displaying dates.
+
+You can set PHP's time zone setting with `.user.ini`. For instance, to use the french time zone, edit `.user.ini` to add this line:
```
Comment (urcadox) [inline runtimes/php/php-apps.md]: ... can I insist that you add a "However" or something at the beginning? :)

Comment (divarvel) [reply runtimes/php/php-apps.md]: I've added "additionally"

**Pull request merged**